src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -32,11 +36,33 @@ const linkToCourse = (course, slug) => {
   return `/${slug}/course/${course.key}?${qs.stringify(queryParams)}`;
 };
 
+const renderDialog = () => (
+  <div className="lead d-flex align-items-center py-3">
+    <div className="mr-3">
+      <FontAwesomeIcon icon={faSearchMinus} size="2x" />
+    </div>
+    <p>
+      { NO_COURSES_ALERT_MESSAGE }
+    </p>
+  </div>
+);
+
 const SearchCourseCard = ({ index }) => {
-  const { enterpriseConfig: { slug } } = useContext(AppContext);
+  const { enterpriseConfig } = useContext(AppContext);
```

Comment:
Can't we access it as it was accessible earlier?
`const { enterpriseConfig: { slug } } = useContext(AppContext);`

Reply:
We need `enterpriseConfig` as well, that's why I am destructuring `slug` in the next statement.